.github/workflows/main.yml

```diff
@@ -0,0 +1,66 @@
+# This is a basic workflow to help you get started with Actions
```

Comment:
If that from a template you have used? maybe we can remove this line

Reply:
Removed the comments.